### ISSUE TITLE
Teamtalk android. There is 2 things that has been made incorrectly, users focuses and channel focuses

### ISSUE THREAD

**Issue by HanaDadfar** [BearWare/TeamTalk5#2146]
### Description

if you enable your talkback's sound feedback and vibration feedback, you will see sometimes talkback has a difrent sound and vibration for difrent focuses, there are 2 types of them. Read only focus and clicable focus. then, if you go to teamtalk and focus on a user or focus on a channel, you will see that talkback showes the focus as a read only object, but, it should be clicable objects. for example, compare the msg button's focusds, and user's focus.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer
